backend/db_setup/schema.indaba.sql

```diff
@@ -6937,6 +6963,10 @@ ALTER TABLE ONLY "Tasks"
     ADD CONSTRAINT "Tasks_userId_fkey" FOREIGN KEY ("userId") REFERENCES "Users"(id);
 
 
+-- TODO: Delete this if we end up not needing it. Doesn't seem like we should have this constraint
```

Comment:
Let me know if you still thing we should have this constraint and I will uncomment this. Otherwise, I will delete it before we merge. 

I don't think we should have this constraint. It feels weird to limit this column to only items in `ProjectUsers`. There is already a constraint for the `UserId` column in the `Tasks` table which says that all user ID's must exist in the `User` table.

Reply:
Then no, we can drop that constraint.

Reply:
If we have some other constraint taking care of that for us, then feel free to remove this constraint. I appreciate constraints, but really don't want to play the unraveling game whenever I need to manually alter data.